WalletWasabi.Fluent/ViewModels/Wallets/Settings/ExcludedCoinsViewModel.cs

```diff
@@ -0,0 +1,75 @@
+using System.Linq;
+using System.Reactive;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using System.Windows.Input;
+using DynamicData;
+using DynamicData.Aggregation;
+using DynamicData.Binding;
+using ReactiveUI;
+using WalletWasabi.Fluent.Infrastructure;
+using WalletWasabi.Fluent.Models.Wallets;
+using WalletWasabi.Fluent.ViewModels.Dialogs.Base;
+using WalletWasabi.Fluent.ViewModels.Wallets.Coins;
+
+namespace WalletWasabi.Fluent.ViewModels.Wallets.Settings;
+
+[AppLifetime]
```

Comment:
I don't see why is it AppLifetime. This instance will be created on navigation every time.